frontend/js/animation.js

```diff
@@ -41,8 +41,9 @@ export default function initAnimation(port) {
           maxDistance = 150,
           minDistance2 = minDistance * minDistance,
           maxDistance2 = maxDistance * maxDistance;
+
       var tau = 2 * Math.PI,
-          n = 20,
+          n = Math.max(Math.floor(width*height/15000), 20),
```

Comment:
Put spaces around operators, and to my eyes divided by 25000 looks more pleasing. This amount is a bit much.